homeassistant/components/light/osramlightify.py

```diff
@@ -198,6 +196,8 @@ def turn_on(self, **kwargs):
             self._luminary.set_luminance(
                 int(self._brightness / 2.55),
                 transition)
+        else:
+            self._luminary.set_onoff(1)
```

Comment:
This makes the preceding `set_rgb` and `set_temperature` calls happen while the light is still off. Does that still work?